### PR TITLE
scrollableColumnIndex  and columnInfo.bodyScrollableColumn fix.

### PR DESCRIPTION
**Pull request by mfyuce**
"columnInfo.bodyScrollableColumns.length" is "0" then "columnInfo.bodyScrollableColumns.length - 1"  becomes "-1" and this index is not accepted

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: dist/fixed-data-table-base.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * FixedDataTable v0.8.6 
+ * FixedDataTable v0.8.8
```
Comment (wcjordan) [inline dist/fixed-data-table-base.css]: We don't want to modify these files directly.  Could you revert your changes to the dist directory for the PR?

File: src/FixedDataTable.js [hunk 2]
```diff
@@ -1239,9 +1239,9 @@ var FixedDataTable = createReactClass({
         var availableScrollWidth = adjustedWidth - totalFixedColumnsWidth;
 
         // Get width of specified column
-        var selectedColumnWidth = columnInfo.bodyScrollableColumns[
+        var selectedColumnWidth =columnInfo.bodyScrollableColumns.length >0? columnInfo.bodyScrollableColumns[
```
Comment (wcjordan) [inline src/FixedDataTable.js]: spacing seems incorrect here, could you fix?

File: src/FixedDataTable.js [hunk 3]
```diff
@@ -1239,9 +1239,9 @@ var FixedDataTable = createReactClass({
...
           scrollableColumnIndex
-        ].props.width;
+        ].props.width:100;
```
Comment (wcjordan) [inline src/FixedDataTable.js]: Does the value used here matter?  If not can you use 0 instead of 100 to avoid magic numbers.

Comment (mfyuce) [reply src/FixedDataTable.js]: Hi, 

I think it is an edge case; when there is no state ( and so no columns etc) and then indicating scrollable columns.

I have added an if statement at the beginning of the block (if it is OK).

Spacing and the magic number should be OK, as I removed them.

Thanks for the great work.

File: src/FixedDataTable.js [hunk 4]
```diff
@@ -1204,7 +1204,9 @@ var FixedDataTable = createReactClass({
     );
 
     var lastScrollToColumn = oldState ? oldState.scrollToColumn : undefined;
-    if (props.scrollToColumn !== null && props.scrollToColumn !== lastScrollToColumn) {
+    if (props.scrollToColumn !== null 
+        && props.scrollToColumn !== lastScrollToColumn
+        && columnInfo.bodyScrollableColumns.length>0) {
```
Comment (wcjordan) [inline src/FixedDataTable.js]: Can you add spaces around the > operator to fit our styles?

After that, I'll merge and cut a point release.  Thanks

**Comment by mfyuce** [comment]
Just pushed another commit for the style fix. 

Thanks for the good work.

**Comment by wcjordan** [comment]
thanks

**Comment by wcjordan** [comment]
Released w/ v0.8.8

**Pull request merged**